README.md

```diff
@@ -1,23 +1,27 @@
-# Riskbook's clone of haskell-xmpp
+[![Maintenance badge](https://img.shields.io/maintenance/yes/2020)](https://riskbook.com/)
 
-Fixes:
+Fully functional haskell-xmpp bindings used in production
+at riskbook.
+This integrates well with ejabbered for example.
+
+Haskell XMPP (eXtensible Message Passing Protocol, a.k.a. Jabber) library.
+Unlike package network-protocol-xmpp, which uses libxml-sax, this library uses HaXml and supports MUC.
+However, MUC support of the moment is worse than that in package XMPP.
+This library make extensive use of STM and threads to simplify writing message-handling code.
+
+As of version 2 riskbook maintains this.
+We patched the orignal work by dmitry with these fixes:
 - [x] Duplicate Show instances
 - [x] Missing FlexibleContexts
 - [x] GADTs for Stanzas
 
-Original README contents follow:
-*** 
-Attempt to write an XMPP bindings for haskell
-
-Alpha-version/work in progress
+## Contributing
+Feel free to submit any PR or issue.
 
-To build, obtain and install latest HaXml 
-(darcs get http://www.cs.york.ac.uk/fp/darcs/HaXml)
-, then use supplied rebuild.sh
+Questions can be send to: jappie@riskbook.com
```

Comment:
Maybe use Riskbook shared email instead?

Reply:
On the one hand I agree with you it's better to have the company email catch this, but on the other hand I'd prefer to speak to a person than an entire company for questions.

Reply:
It is a standardised technique to add something like `tech@company.com` so the entire tech team can communicate, regardless of the availability of only one person. Also maybe some other people in the team would like to be in the loop, so you should at least ask the team who wants to be in this list and also frequently update it with newcomers or remove the ones who leave the company 🙂